packages/web-components/src/components/cbp-structured-list/cbp-structured-list.tsx

```diff
@@ -45,21 +45,22 @@ export class CbpStructuredList {
       ...this.sx,
     });
   }
-
+//todo: wrap the header & footer in a div to keep seperate form the generric slot
   render() {
     return (
       <Host>
-        <slot name="cbp-structured-list-header" />
+        <slot name="cbp-structured-list-header" /> 
  
         <div role="list"
           aria-label={this.accessibilityText}
           aria-describedby={this.headerId}
         >
           <slot />
         </div>
- 
-        <slot name="cbp-structured-list-footer" />
-      </Host>
+        <div class='cbp-structured-list-footer'>
```

Comment:
if you're not actually targeting the class name in your CSS, then you don't need it. Do you need the wrapping div at all still? I don't recall the reasoning after realizing the side effects now. Wasn't it just so that you're not applying some CSS to the action bar and doubling up on padding?

Reply:
updates as per call